v3/common/type_test.go

```diff
@@ -261,6 +285,24 @@ func TestTranslateStringToType__TypeLong(t *testing.T) {
 	}
 }
 
+func TestTranslateStringToType__TypeFloat(t *testing.T) {
+	v, err := TranslateStringToType(TypeFloat, "3.14159265")
+	log.PanicIf(err)
+
+	if v != float32(3.1415927410125732) {
```

Comment:
Use [Nextafter32](https://golang.org/pkg/math/#Nextafter32) and [Nextafter](https://golang.org/pkg/math/#Nextafter). You shouldn't compare floats directly.